src/main/java/Duke.java

```diff
@@ -1,10 +1,240 @@
+import java.util.Random;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
 public class Duke {
+    private static File save;
+
+    public static String addTaskSalutation(String item) {
+        Random rand = new Random();
+        String out = "";
+        switch(rand.nextInt(3)) {
+        case 0:
```

Comment:
You are using integers for the switch cases. How about creating more meaningful names for each case?